exporter/lokiexporter/logproto/logproto.pb.go

```diff
@@ -2492,7 +2492,7 @@ func (this *SeriesIdentifier) GoString() string {
 	s := make([]string, 0, 5)
 	s = append(s, "&logproto.SeriesIdentifier{")
 	keysForLabels := make([]string, 0, len(this.Labels))
-	for k, _ := range this.Labels {
```

Comment:
Why do we need to change the exporter?

Reply:
There is something wrong with the code format of this exporter, I just ran `make fmt`, this one changed.

Reply:
I don't think we can revise this part

Reply:
Will delete this part, it seems that someone else changed it, makes it a conflicting file.